### ISSUE TITLE
Fonts not exporting correctly to PNG

### ISSUE THREAD

**Issue by Geoff332** [jgraph/drawio#693]
I'm attempting to create diagrams using formatted text using drawio in google drive, using google fonts. When I export these images to PNG, it sometimes loses the font and reverts to a default serif font.

This is the drawing in drawio:
![image](https://user-images.githubusercontent.com/24893354/71414780-690e3780-2650-11ea-9fa8-1b6271d4e8a7.png)

And the font settings for one of the text elements:
![image](https://user-images.githubusercontent.com/24893354/71414802-8f33d780-2650-11ea-8ad5-85400cc68e8d.png)

The export settings:
![image](https://user-images.githubusercontent.com/24893354/71414822-a96db580-2650-11ea-8236-12f208c5eb0b.png)

And the exported image:
![Roadmaps-Backlog](https://user-images.githubusercontent.com/24893354/71414884-f18cd800-2650-11ea-9f24-59f71d59cb98.png)

In some of my diagrams, this works; in others it doesn't. In this case, the text element I'm using was copied from another sheet in the same diagram which exported correctly.

**Comment by davidjgraph** [comment]
Using what browser?

**Comment by Geoff332** [comment]
Using Chrome.

I managed to get it working; it looks like the system fonts vs others are getting tangled up. If I select Monserrat from this list, it displays correctly in wysiwyg in Chrome, but doesn't recognise it as a system font on export.
![image](https://user-images.githubusercontent.com/24893354/71520043-2481d680-28b2-11ea-9cc5-4a84a2e7c6d2.png)

If I open the custom font settings, then I can remove the system font and explicitly define it as a Google font:
![image](https://user-images.githubusercontent.com/24893354/71520069-424f3b80-28b2-11ea-9c7b-2c30cccc431b.png)

And the export now works (I didn't figure this out last week! If nothing else, knowing how to make sure it works is helpful). It's quite likely I defined Monserrat as a system font at some point, before fixing it.
